src/router/index.js

```diff
@@ -1,19 +1,62 @@
-import React from "react";
-import {Routes, Route, Navigate} from "react-router-dom";
+import React, {useState} from "react";
+import {Routes, Route} from "react-router-dom";
 
 import GettingStarted from "Components/GettingStarted";
 import Components from "Components/Components";
-import Fabric from "Components/Fabric";
+import Home from "Components/Home";
+import Navbar from "Components/Navbar";
+import { ElvWalletClient } from "@eluvio/elv-wallet-client"; 
+
+
+
+const AppRoutes = React.memo(() => {
+
+  const [profile, setProfile] = useState({});
+  const [loggedIn,setLoggedIn] = useState(false);
```

Comment:
Can you move `loggedIn` state to a data store? We don't have any set up yet but the structure should be src/stores/index.js (rootStore). Look at elv-music-js's rootStore and elv-media-wallet's rootStore